Vineyard/Vineyard/GroupsListViewModel.swift

```diff
@@ -14,6 +14,7 @@ class GroupsListViewModel: ObservableObject {
     var groups: [Group] = []
     var isPresentingCreateGroupView = false
     var isPresentingCreateGoalView = false
+    var isPresentingGoalsListView = false
```

Comment:
What does this do? Not seeing it anywhere else unless im missing something again

Reply:
removed